src/client/components/Delete_Comfim/ModalList.component.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Button from '../button/button.component';
+
+const ModalList = ({ isShowing, hide, onClick }) => {
+  if (isShowing) {
```

Comment:
You don't actually need to check this here, since `isShowing` is defined in the `Button` component. You can check over there whether `isShowing` is true and decide whether or not to render `ModalList` (see the last comment I left in the `Button` component file...).